src/components/Card/Card.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import './assets'
```

Comment:
Try to import assets correctly, wrong import.
![image](https://github.com/siddharth17vaishnav/viewCommerce/assets/73464543/055750fa-4809-46bb-9c14-5389ba90ce87)